bin/cml/pr.js

```diff
@@ -19,10 +19,21 @@ exports.builder = (yargs) =>
         type: 'boolean',
         description: 'Output in markdown format [](url).'
       },
-      autoMerge: {
+      merge: {
         type: 'boolean',
-        description:
-          'Mark the PR/MR for automatic merging after tests pass (unsupported by Bitbucket).'
+        alias: 'auto-merge',
+        conflicts: ['rebase', 'squash'],
+        description: 'Try to merge the pull request upon creation.'
```

Comment:
technically on CI passing (GH & GL) and on creation (BB)

Reply:
It's auto–merge with fallback to direct merge on GitHub and GitLab, and direct merge only on Bitbucket.

Reply:
The general description might be “best effort merge” on every provider

Reply:
(Feel free to open a follow–up pull request with a more accurate description)